crates/bevy_render/src/camera/projection.rs

```diff
@@ -255,11 +255,14 @@ pub struct OrthographicProjection {
     ///
     /// Defaults to `ScalingMode::WindowSize(1.0)`
     pub scaling_mode: ScalingMode,
-    /// Scales the projection in world units.
+    /// Scales the projection.
     ///
     /// As scale increases, the apparent size of objects decreases, and vice versa.
     ///
-    /// Defaults to `1.0`
+    /// Note: scaling can be set by [`scaling_mode`](Self::scaling_mode) as well.
+    /// This parameter scales on top of that.
+    ///
```

Comment:
https://github.com/bevyengine/bevy/pull/11022#issuecomment-1865106453 mentioned that it would be better to mention `scale`'s utility in implementing zoom.
```suggestion
    ///
    /// This property is particularly useful in implementing zoom functionality.
    ///
```